docs/data/tree-view/rich-tree-view/selection/selection.md

```diff
@@ -11,12 +11,32 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
 
 <p class="description">Handle how users can select items.</p>
 
+## Single selection
+
+By default, the Tree View allows selecting a single item.
+
+{{"demo": "SingleSelectTreeView.js"}}
+
+:::success
+When the Tree View uses single selection, you can select an item by clicking it,
+or using the [keyboard shortcuts](/x/react-tree-view/accessibility/#on-single-select-trees).
+:::
+
 ## Multi selection
 
-The Tree View also supports multi-selection:
+Use the `multiSelect` prop to enable multi-selection.
 
 {{"demo": "MultiSelectTreeView.js"}}
 
+:::success
```

Comment:
Unlike the data grid, I added those as `success` because IMHO this is more a side not than the main content here.
The main content is how to enable selection, and then we describe how to use selection but this mostly target end users which won't read to doc, so it's more to say "Hey we have good selection interactions".